### PR TITLE
User

### PR DESCRIPTION
**Pull request by lohoi**

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def new
+  end
+
+  def create
+    # user = User.find_by(email: params[:session][:email].downcase)
```
Comment (RemingtonReackhof) [inline app/controllers/sessions_controller.rb]: Delete comments

File: app/controllers/sessions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    #     log_in user
+    #     format.html { redirect_to user }
+    #   else
+    #     format.html { render :new, notice: "Login failed, invalid email/password combination" }
+    #   end
+    # end
+  end
+
+  def destroy
+    # log_out if logged_in?
```
Comment (RemingtonReackhof) [inline app/controllers/sessions_controller.rb]: Delete comments, do we need the functions?

File: config/database.yml [hunk 3]
```diff
@@ -15,7 +15,8 @@ default: &default
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   username: root
   password: root
-  socket: /tmp/mysql.sock
+  socket: /var/run/mysqld/mysqld.sock
```
Comment (RemingtonReackhof) [inline config/database.yml]: remove from commit

File: test/controllers/sessions_controller_test.rb [hunk 4]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class SessionsControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```
Comment (RemingtonReackhof) [inline test/controllers/sessions_controller_test.rb]: remove comments?

File: web-app/src/app/user.service.ts [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+      return this.authService.currentUserData["user_type"] === "doctor"
+    }
+  }
+
+  getUser(): any {
+    return this.authService.currentUserData;
+  }
+
+
+ //  login = function(email_, password_) {
```
Comment (RemingtonReackhof) [inline web-app/src/app/user.service.ts]: remove comments

File: web-app/src/app/user.ts [hunk 6]
```diff
@@ -0,0 +1,10 @@
+// user class
+export class User {
+  	email:string
```
Comment (RemingtonReackhof) [inline web-app/src/app/user.ts]: fix tabbing

File: web-app/src/environments/environment.prod.ts [hunk 7]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (RemingtonReackhof) [inline web-app/src/environments/environment.prod.ts]: remove newline

File: web-app/src/environments/environment.ts [hunk 8]
```diff
@@ -4,5 +4,13 @@
 // The list of which env maps to which file can be found in `angular-cli.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  token_auth_config: {
+    apiBase: 'http://localhost:3000',
+    signInPath: 'auth/sign_in',
+    // userTypes: [
```
Comment (RemingtonReackhof) [inline web-app/src/environments/environment.ts]: remove comments

**Review by RemingtonReackhof — CHANGES_REQUESTED** [review]
@lohoi remove the commented out code and then we should be good

File: config/database.yml (hunk 3 above)
Comment (lohoi) [reply config/database.yml]: done

File: test/controllers/sessions_controller_test.rb (hunk 4 above)
Comment (lohoi) [reply test/controllers/sessions_controller_test.rb]: done

File: web-app/src/environments/environment.prod.ts (hunk 7 above)
Comment (lohoi) [reply web-app/src/environments/environment.prod.ts]: done

File: web-app/src/environments/environment.ts (hunk 8 above)
Comment (lohoi) [reply web-app/src/environments/environment.ts]: done

**Pull request merged**